brave/brave-core
# fix(iOS): Add 1.71 translation

Resolves https://github.com/brave/brave-browser/issues/42110
Resolves https://github.com/brave/brave-browser/issues/41731



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. change device language to a language other than english
2. open Brave and check if all screens are localized correctly 
3. make sure Sync screens are localized
